kotlinx-coroutines-core/common/src/flow/Flow.kt

```diff
@@ -172,7 +172,7 @@ import kotlin.coroutines.*
  *
  * **The `Flow` interface is not stable for inheritance in 3rd party libraries**, as new methods
  * might be added to this interface in the future, but is stable for use.
- * Use the `flow { ... }` builder function to create an implementation.
+ * Use the `flow { ... }` builder function to create an implementation or extend [AbstractFlow].
```

Comment:
This parses as "Use the `flow { ... }` builder function to (create an implementation or extend [AbstractFlow])". I guess you probably meant
```suggestion
 * Use the `flow { ... }` builder function to create an implementation, or extend [AbstractFlow].
```
```suggestion
 * Use the `flow { ... }` builder function or extend [AbstractFlow] to create an implementation.
```